requirements-dev.in

```diff
@@ -13,6 +13,7 @@ playwright
 requests
 types-requests
 whitenoise
+pydantic
```

Comment:
Updated dev deps and tox deps to include pydantic, so I could add a test for how the component error handling behaves with Pydantic's ValidationError.